data.json

```diff
@@ -1,4 +1,10 @@
 [
+    {
+        "name": "Savio Martin",
+        "githubUsername": "saviomartin",
+        "imageUrl": "https://raw.githubusercontent.com/saviomartin/saviomartin/master/assets/saviomartin.png",
+        "issueNumber": 10
```

Comment:
Issue Number 10 is https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/10
I see that you have raised an issue:
https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/98
Shouldn't this be 98?

Reply:
**Oh Sorry**, I didn't think like that. I just took a number.
**I  have updated it.** 

Is that enough?
thankyou